bundles/io/org.eclipse.smarthome.io.transport.mdns/OSGI-INF/metatype/org.eclipse.smarthome.io.transport.mdns.internal.MDNSClientImpl.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.osgi.org/xmlns/metatype/v1.1.0 http://www.osgi.org/xmlns/metatype/v1.1.0">
+
+	<OCD name="Transport mDNS" id="TransportMDNS" description="Transport mDNS IP configuration">
+		<AD name="useOnlyOneAddress" id="useOnlyOneAddress" default="false" type="Boolean" description="Use only one IP address per interface and family" />
```

Comment:
Should default to true in my opinion

Reply:
I didn't want to change the default behaviour.